compiler/one-cmds/tests/one-infer_006.py

```diff
@@ -0,0 +1,14 @@
+# This script gets one argument and print it
+
+import sys
+from pathlib import Path
+
+def main():
+    if len(sys.argv) < 2:
+        filepath = Path(sys.argv[0])
+        sys.exit("Usage: " + filepath.name + " [Word to print]")
+    word = sys.argv[1]
+    print(word)
+
+if __name__ == '__main__':
+    main()
```

Comment:
optional) how about return with value, or exit with a value so that caller can check with it for success or failure.

Reply:
I'm a little bit confused about this comment. Which part do you want to return with a value? 
- Adding a process exit code to check process' succeed? Or 
- add return value to `main` function to check main function's validity?

Reply:
Well, as I'm not good at python, what I thought was like this called from bash or other process
```
python3 one-infer_006.py
ret = $?
echo $ret
# check ret value
```
Could you please how process exit code or return value of main shows?

Reply:
> Could you please how process exit code or return value of main shows?

I'm sorry I couldn't catch what you saying.. :sob:

Current `one-infer_006.py` prints the first given argument if it successfully executed. 
To check whether a process which executes `one-infer_006.py` as its subprocess, we can check the printed line includes the first argument. (As you know, it's how the `one-infer_006.test` works). I'm not sure how to make test codes for both options you mentioned, exit code or return value.

Reply:
I got your intention through the off-line talk. :wink:
Since `python3 some_python_script` will emit exit code as zero if the script ended successfully, and non-zero for else, it would be better to check its exit code for both valid and invalid post processing script.

`one-infer -d dummy-infer --post-process "[valid-command]" -- [infer command]`'s exit code will be zero.
`one-infer -d dummy-infer --post-process "[invalid-command]" -- [infer command]`'s exit code will be non-zero.

One positive test and one negative test will be added. :smile: